.github/workflows/blade-pr-title-check.yml

```diff
@@ -0,0 +1,40 @@
+name: Blade PR Title Check
+
+on: 
+  pull_request:
+     types: [open, edited, synchronize]
```

Comment:
Need a separate workflow for this because the trigger types are different. If we keep the same workflow as `blade-validate`, rest of the validate jobs will get triggered even if the PR title is changed